### ISSUE TITLE
[FEATURE] "Play button" in cron UI to test if the cron will run.

### ISSUE THREAD

**Issue by jlguerrerosanz** [hestiacp/hestiacp#2020]
There is no way to know if I misspelled the cron command until the time arrives.

A  Play button     >    to execute the command right away and see the output or at least to know if there were some errors would be handy.

![image](https://user-images.githubusercontent.com/8811720/127295118-16d158c7-00a7-4480-ac05-ff81faa4328b.png)

**Comment by jlguerrerosanz** [comment]
There could also be the possibility to capture the output of the command and show it.
Or at least show if there have been error codes after some time of execution.

**Comment by zikzakplus** [comment]
I agree. This would be very helpful. I am having problems running some of my corn jobs and I can't figure out why. I can run them when SSH but the corn job doesn't work. I have three cron jobs and two of them don't run.

**Comment by killserver** [comment]
> There could also be the possibility to capture the output of the command and show it. Or at least show if there have been error codes after some time of execution.

maybe show in modal result execution cron job?)
write result to temp file and in web - get result from file after timeout.. )

**Comment by thytetgc** [comment]
It would be great to have Cron in the GUI as well as in aaPanel...
